.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  extends: ["./index.js"],
+  overrides: [
+    {
+      files: ["packMain.js"],
+      parser: "esprima",
```

Comment:
what is this custom parser for?

Reply:
It's not a custom one, it's the default they use for javascript. We use typescript parser, but for files that are not in `tsconfig` you need to switch to a different one otherwise, the typescript parser will detect that the file is not part of the project.

Reply:
Can you make it in a way we don't need to add specific files here as we add them in the project?

Reply:
Not sure I follow. It's a project-specific file that does not fall into the typescript project.
Where would you configure it if not in the project-specific linter setup?

Reply:
I mixed up things. This file is only used in the eslint-rules-pkg and extends the configuration we expose. It is not part of the config that the projects will consume